spec/controllers/admin/user_creations_controller_spec.rb

```diff
@@ -164,4 +164,151 @@
       it_behaves_like "unauthorized admin cannot delete bookmarks"
     end
   end
+
+  shared_examples "an action only authorized admins can access" do |authorized_roles:|
```

Comment:
Looks like this block is defined in a bunch of places. Can you move it into `spec/support/shared_examples/admin_shared_examples.rb`?